homework02/exercise_6.py

```diff
@@ -0,0 +1,109 @@
+'''
+Edit this file to complete Exercise 6
+'''
+
+def calculation(a, b):
+	'''
+	Write a function calculation() such that it can accept two variables 
+	and calculate the addition and subtraction of it. 
+	It must return both addition and subtraction in a single return call
+
+	Expected output:
+	res = calculation(40, 10)
+	print(res)
+	>>> (50, 30)
+
+	Arguments:
+	a: first number 
+	b: second number
+
+	Returns:
+	sum: sum of two numbers
+	diff: difference of two numbers
+	'''
+
+	# code up your solution here
+    return (a+b, a-b)
```

Comment:
make sure to understand this return is a list, you can also return two values directly